apps/backoffice-v2/src/domains/auth/hooks/mutations/useSignOutMutation/useSignOutMutation.tsx

```diff
@@ -18,9 +18,7 @@ export const useSignOutMutation = () => {
       void queryClient.cancelQueries();
     },
     onSuccess: (data, { callbackUrl, redirect }) => {
-      queryClient.setQueryData(authenticatedUser.queryKey, {
-        user: undefined,
-      });
+      void queryClient.invalidateQueries();
```

Comment:
When the user logs out - it should invalidate all of its cache, including workflows, etc.